vitest.config.ts

```
@@ -0,0 +1,19 @@
+import { defineConfig } from "vitest/config";
+import path from "path";
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)